File: 01.JS-Basic/More Exercises/PB - More Exercises/For-Loop/Clock.js
function clock() {

    let minutes = 0
    let hour = 0

    for (let i = 0; i < 1440; i++) {

        if (minutes == 60) {
            hour++
            minutes = 0
        }
        console.log(`${hour} : ${minutes}`)

        minutes++

    }
}
clock()

File: JAVA-Basic/JavaBasic/src/NestedLoops/Clock.java
package NestedLoops;

public class Clock {
    public static void main(String[] args) {

        for (int i = 0; i <= 23; i++) {
            for (int x = 0; x <= 59; x++) {
                System.out.println(i + ":" + x);
            }
        }
    }
}
